0.0.1/src/utils.py

```diff
@@ -396,24 +397,52 @@ async def main():
 #
     # publish_payload = sign_quote(payload)
 
-    # TODO test client methods
     # client = NearMpcClient(network="mainnet")
 
     # client.derive_mpc_key("agent.charleslavon.near")
 
     # signed_intent = await client.sign_intent("agent.charleslavon.near", best_quote["token_in"], best_quote["token_out"], best_quote["amount_in"], best_quote["amount_out"], best_quote["quote_hash"], best_quote["expiration_time"], nonce_base64)
 ##
     # publish_payload = Commitment(
-    #    standard="raw_ed25519", ## TODO start here, what standard to use?
+    #    standard="raw_ed25519",
     #    payload=json.dumps(signed_intent.get("intent")),
     #    signature=signed_intent.get("signature"),
     #    public_key=signed_intent.get("public_key")
     # )
 #
-    publish_payload = PublishIntent(signed_data=publish_payload, quote_hashes=[
-                                    best_quote.get("quote_hash")])
+    # publish_payload = PublishIntent(signed_data=publish_payload, quote_hashes=[
+#                                    best_quote.get("quote_hash")])
 
-    print(publish_intent(publish_payload))
+    # print(publish_intent(publish_payload))
 
 
-# asyncio.run(main())
+async def demo_quote():
+    """Demonstrate OneClick API quote functionality"""
```

Comment:
When do we do this demo, when testing?

Reply:
Yea I added this method to test these functions manually with ‘python manual_tests.py’